### ISSUE TITLE
LLVM frontend

### ISSUE THREAD

**Issue by graingert** [jamesalbert/HolyC-for-Linux#1]
Support Linux by writing an LLVM frontend

See https://www.reddit.com/r/programming/comments/6290hb/_/dfm6nmr

**Comment by jamesalbert** [comment]
I agree, I'm looking into it right now. I think this will make it significantly more maintainable than the current state it's in. Thanks for the issue
